spec/features/consumer/shopping/cart_spec.rb

```diff
@@ -56,6 +56,7 @@
       let(:variant) { product_tax.variants.first }
 
       before do
+        Spree::Config.set allow_backorders: false
```

Comment:
This is super easy to modify.

But I left it as it is as it feels a bit controversial as it seems that business wants `allow_backorders: false` all the time. And we were facing many issues and wasting a lot of time figuring this out. Also, if you would scan all specs you would find more settings override like this. @oeoeaio, @mkllnk, @lin-d-hop, maybe worth more of discussion?